interface/patient_file/summary/pnotes_full.php

```diff
@@ -276,469 +276,462 @@ function restoreSession() {
+                    <?php if (!$orderid) { ?>
+                        <a href="demographics.php" class="btn btn-secondary btn-back" onclick="top.restoreSession()"><?php echo xlt('Back to Patient'); ?></a>
+                    <?php } ?>
+                    <a href="pnotes_full.php?<?php echo $urlparms; ?>&<?php echo $activity_string_html;?>" class="btn btn-secondary btn-update" id='Submit' onclick='return top.restoreSession()'><?php echo xlt('Refresh'); ?></a>
+                    <a href="#" class="change_activity btn btn-secondary btn"><?php echo xlt('Update Active'); ?></a>
+            </div>
+            </div>
+
         </div>
+        <div class="row oe-margin-b-10">
+            <div class="col-12">
+                <?php
+                // Get the billing note if there is one.
+                $billing_note = "";
+                $colorbeg = "";
+                $colorend = "";
+                $resnote = getPatientData($patient_id, "billing_note");
+                if (!empty($resnote['billing_note'])) {
+                    $billing_note = $resnote['billing_note'];
+                    $colorbeg = "<span class='text-danger'>";
+                    $colorend = "</span>";
+                }
 
-    </div>
-    <div class="row oe-margin-b-10">
-        <div class="col-md-12">
-            <?php
-            // Get the billing note if there is one.
-            $billing_note = "";
-            $colorbeg = "";
-            $colorend = "";
-            $resnote = getPatientData($patient_id, "billing_note");
-            if (!empty($resnote['billing_note'])) {
-                $billing_note = $resnote['billing_note'];
-                $colorbeg = "<span style='color:red'>";
-                $colorend = "</span>";
-            }
+                //Display what the patient owes
+                $balance = get_patient_balance($patient_id);
+                ?>
+
+                <?php if ($billing_note || $balance) { ?>
+                    <div class="mt-1">
+                        <table class="table">
+                            <?php
+                            if ($balance != "0") {
+                                // $formatted = sprintf((xl('$').'%01.2f'), $balance);
+                                $formatted = oeFormatMoney($balance);
+                                echo " <tr class='text billing'>\n";
+                                echo "  <td>" . $colorbeg . xlt('Balance Due') .
+                                    $colorend . "&nbsp;" . $colorbeg . text($formatted) .
+                                    $colorend . "</td>\n";
+                                echo " </tr>\n";
+                            }
+
+                            if ($billing_note) {
+                                echo " <tr class='text billing'>\n";
+                                echo "  <td>" . $colorbeg . xlt('Billing Note') .
```

Comment:
Hi @tywrenn 

I am not sure, but I guess the `.text` the class has been deprecated, Should I remove it?

Reply:
That actually does a font resize, found in `/interface/themes/core/text.scss`. It isn't deprecated *yet*.